src/sas/qtgui/Calculators/media/shape_2_sas_help.rst

```diff
@@ -0,0 +1,87 @@
+| 2           | ΔSLD             | The SLD of the particle relative to the medium surrounding it. Defaults to 1.0                       |
++-------------+------------------+------------------------------------------------------------------------------------------------------+
+| 3, 4, and 5 | Size Definitions | The sizes for each major axis of the particle, in Å. The actual values will vary based on the shape. |
++-------------+------------------+------------------------------------------------------------------------------------------------------+
+| 6           | COMX             | The relative offset along the X axis of the particle centroid from the coordinate origin, in Å.      |
++-------------+------------------+------------------------------------------------------------------------------------------------------+
+| 7           | COMY             | The relative offset along the Y axis of the particle centroid from the coordinate origin, in Å.      |
++-------------+------------------+------------------------------------------------------------------------------------------------------+
+| 8           | COMZ             | The relative offset along the Z axis of the particle centroid from the coordinate origin, in Å.      |
++-------------+------------------+------------------------------------------------------------------------------------------------------+
+| 9           | RPX              | The center of rotation of the particle, along the X axis, relative to the coordinate origin, in Å.   |
++-------------+------------------+------------------------------------------------------------------------------------------------------+
+| 10          | RPY              | The center of rotation of the particle, along the Y axis, relative to the coordinate origin, in Å.   |
++-------------+------------------+------------------------------------------------------------------------------------------------------+
+| 11          | RPZ              | The center of rotation of the particle, along the Z axis, relative to the coordinate origin, in Å.   |
++-------------+------------------+------------------------------------------------------------------------------------------------------+
+| 12          | α                | The rotation of the particle, around the X axis, relative to RPX, in degrees.                        |
++-------------+------------------+------------------------------------------------------------------------------------------------------+
+| 13          | β                | The rotation of the particle, around the Y axis, relative to RPX, in degrees.                        |
++-------------+------------------+------------------------------------------------------------------------------------------------------+
+| 14          | γ                | The rotation of the particle, around the Z axis, relative to RPX, in degrees.                        |
```

Comment:
This does not render in sphinx as far as I can tell (and does not from pull request installer running on my windows 11 machine).  I think you need to use the:
```rst
.. list-table:: Title
   :widths: 25 25 50
   :header-rows: 1

   * - Heading row 1, column 1
     - Heading row 1, column 2
     - Heading row 1, column 3
   * - Row 1, column 1
```